### PR TITLE
Modified Test Cases for Assignment-5 for precise display property.

### PR DESCRIPTION
**Pull request by mayankrajendrat**
In previous test cases, the content was checked for top, left alignment w.r.t body, as the students approaches different mechanisms to align content at the center, we made test cases more generic.

### CODE REVIEW

File: src/Assignment-5/assignment5_test.js [hunk 1]
```diff
@@ -1,125 +1,78 @@
 // Copyright (c) 2015 Titanium I.T. LLC. All rights reserved. For license, see "README" or "LICENSE" file.
 
 // Example Quixote unit tests. We're using Mocha as our test framework and Chai for assertions.
-// These tests check the our media object CSS, which is defined in `screen.css`.
+// These tests check the our media object CSS, which is defined in `assignment2.css`.
```
Comment (sailesh-ramayanam) [inline src/Assignment-5/assignment5_test.js:4]: @satya-9 Please validate these test cases. I will wait for your confirmation before approving this PR.

Comment (mayankrajendrat) [reply src/Assignment-5/assignment5_test.js:4]: In previous test cases, the content was checked for top, left alignment w.r.t body, as the students approaches different mechanisms to align content at the center, we made test cases more generic.

File: src/Assignment-5/assignment5_test.js [hunk 2]
```diff
@@ -1,125 +1,78 @@
...
-				"<div class='media'>" +
-				// ...a figure...
-				"  <div id='figure' class='media__figure' style='width:100px; height:100px'>figure</div>" +
-				// ...and content.
-				"  <div id='content' class='media__content'>content</div>" +
-				"</div>",
-				// Give the HTML a name so Quixote's error messages are more readable. If we don't provide a name,
-				// Quixote will use the HTML by default. That would be pretty ugly.
-				"media object"
+	describe("Assignment-5", function () {
```
[the remark this answers is not in the document]
Comment (mayankrajendrat) [reply src/Assignment-5/assignment5_test.js:17]: In previous test cases, the content was checked for top, left alignment w.r.t body, as the students approaches different mechanisms to align content at the center, we made test cases more generic.

[the remark this answers is not in the document]
Comment (sailesh-ramayanam) [reply src/Assignment-5/assignment5_test.js:17]: @mayankrajendrat Please add this information to PR description.

File: src/Assignment-5/assignment5_test.js (hunk 1 above)
Comment (satya-9) [reply src/Assignment-5/assignment5_test.js:4]: @sailesh-ramayanam I tested the testcases using the assignmnet-5 html and css of [https://github.com/wellimaj/assignment-5/tree/assignment-5] github repo and it ran successfully.

Comment (sailesh-ramayanam) [reply src/Assignment-5/assignment5_test.js:4]: @satya-9 Is wellimaj among the students who complained that test cases were not passing?

Comment (satya-9) [reply src/Assignment-5/assignment5_test.js:4]: @sailesh-ramayanam 

1. Yes sailesh.wellimaj is among those students who complained
2. Actually there were no students who passed all the testcases.
3. Yes i checked the solution which was given by mayank and it passed

**Review by sailesh-ramayanam — APPROVED** [review]

**Pull request merged**